landing-panda/src/pages/blogs/blog1.md

```
@@ -0,0 +1,49 @@
+---
+layout: ../blogLayout.astro
+cover: ../../../images/blog1.webp
+---
+# Why I’m Starting This Company
+![Akonga Banner](/images/blog1.webp)
+*adrianmurage*
+*June 14, 2025*
+
+## Outsized Returns
+> *“You’re going to be somewhere in your career in ten years, why not in your absolute ideal role, doing the kind of work you’d find most rewarding and having the kind of impact that you’re capable of? If you don’t believe that this is possible, you will surely fail. If you do believe it’s possible, look honestly — you might still fail! But you’ll at least see better outcomes by believing consistency will work than by believing it won’t.”*
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)